spring-boot-project/spring-boot-actuator-autoconfigure/src/test/java/org/springframework/boot/actuate/autoconfigure/metrics/jdbc/DataSourcePoolMetricsAutoConfigurationTests.java

```diff
@@ -82,7 +83,7 @@ void dataSourceInstrumentationCanBeDisabled() {
 	}
 
 	@Test
-	void allDataSourcesCanBeInstrumented() {
+	void allDataSourceCandidatesCanBeInstrumented() {
```

Comment:
I renamed test names for accuracy, I'll revert this part if you object. @wilkinsona

Reply:
I'd prefer it without `Candidates`, for consistency with all the other tests for similar scenarios if nothing else.